plugins/coupa-review-contract-status/README.md

```diff
@@ -0,0 +1,12 @@
+---
+description: A plugin that allows a permitted employee to review the status of a contract
```

Comment:
Can we pattern match this to other look up plugins?

"Review" implies an approval

Reply:
Meant to add this comment on the plugin name

Reply:
Updating to `Lookup Contract Approval Status from Coupa` since we also have a similar plugin for Ariba

Reply:
Can we actually fix both to be "Look up" -- check the library. That's the correct one.

Lookup is a noun
Look up is a verb phrase